src/test/java/org/opensearch/sdk/TestExtensionsRunner.java

```diff
@@ -142,6 +142,48 @@ public void testHandleExtensionInitRequest() throws UnknownHostException {
         assertEquals(sourceNode, extensionsRunner.getOpensearchNode());
     }
 
+    @Test
+    public void testHandleExtensionInitRequestFromLocalHost() throws UnknownHostException {
+        DiscoveryNode sourceNode = new DiscoveryNode(
+            "test_node",
+            new TransportAddress(InetAddress.getByName("127.0.0.1"), 9876),
+            emptyMap(),
+            emptySet(),
+            Version.CURRENT
+        );
+        DiscoveryExtensionNode extension = new DiscoveryExtensionNode(
+            EXTENSION_NAME,
+            "opensearch-sdk-1",
+            sourceNode.getAddress(),
+            new HashMap<String, String>(),
+            Version.fromString("3.0.0"),
```

Comment:
I think this needs to be changed to `2.8.0`

Reply:
Or any number less than that